apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Blockchain.java

```diff
@@ -65,6 +70,14 @@
 
     DbIterator<Block> getBlocks(long accountId, int timestamp, int from, int to);
 
+    Block findLastBlock();
+
+    Block loadBlock(Connection con, ResultSet rs, boolean loadTransactions);
+
+    void saveBlock(Connection con, Block block);
+
+    void commit(Block block);
+
```

Comment:
Do we really required all these methods here?

Reply:
Yes, we want break dependencies to avoid direct calls to BlockDao/TransactionDao